app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb

```diff
@@ -37,17 +37,9 @@ def connect(options = {})
   end
 
   def api_endpoint_url(location)
-    api_endpoint_overrides = ::Settings.ems.ems_ibm_cloud_power_virtual_servers.api_endpoint_overrides
-
-    if api_endpoint_overrides.key?(location.to_sym)
-      url = api_endpoint_overrides[location.to_sym]
-    else
-      region = location.sub(/-\d$/, '')
-      region = region.sub(/\d\d$/, '')
-      url = "#{region}.power-iaas.cloud.ibm.com"
-    end
-
-    url
+    region = location.sub(/-\d$/, '')  # e.g. 'eu-de-2' -> 'eu-de'
+    region = region.sub(/\d\d$/, '')   # e.g. 'syd04' -> 'syd'
```

Comment:
Actually I think you can do both of these all in one, try `region.sub(/-*\d+$/, '')`
```
>> "eu-de-20".sub(/-*\d+$/, '')
=> "eu-de"
>> "syd0005".sub(/-*\d+$/, '')
=> "syd"
```